profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -306,16 +306,16 @@ function _nexteuropa_multilingual_get_und_entity_alias_info($entity, $entity_typ
     return FALSE;
   }
   // Get the corresponding LANGUAGE_NONE alias.
-  $entity_uri = $entity->path['source'];
-  $language_none_path = path_load(array('source' => $entity_uri, 'language' => LANGUAGE_NONE));
+  $entity_uri = entity_uri($entity_type, $entity);
+  $language_none_path = path_load(array('source' => $entity_uri['path'], 'language' => LANGUAGE_NONE));
   $language_none_alias = $language_none_path['alias'];
 
   // Check if pathauto is activtedin LANGUAGE_NONE case.
   // Logic is taken from Pathauto itself.
   // @see pathauto_entity_presave() & pathauto_field_attach_form().
   module_load_include('inc', 'pathauto');
```

Comment:
```
function _nexteuropa_multilingual_get_und_entity_alias_info($entity, $entity_type) {
  // This function only support entity translation for now.
  list(, , $bundle) = entity_extract_ids($entity_type, $entity);
  $pattern = pathauto_pattern_load_by_entity($entity_type, $bundle, LANGUAGE_NONE);
  if (empty($pattern)) {
    return FALSE;
  }
  // Get the corresponding LANGUAGE_NONE alias.
  $entity_uri = entity_uri($entity_type, $entity);
  $language_none_path = path_load(array('source' => $entity_uri['path'], 'language' => LANGUAGE_NONE));
  $language_none_alias = $language_none_path['alias'];

  // Check if pathauto is activtedin LANGUAGE_NONE case.
  // Logic is taken from Pathauto itself.
  // @see pathauto_entity_presave() & pathauto_field_attach_form().
  module_load_include('inc', 'pathauto');
```

If empty($pattern) it's not necessary to include pathauto.inc. module_load_include is used for pathauto_create_alias, so it's logic that the necessary include is above.